### PR TITLE
Fixed #9.

### LINKED ISSUE #9: Rename `Vsix` class that is generated in manifest to `{FileNamespace}Manifest` instead, and make it `public` not `internal`.
Currently, the `VsixManifestGenerator` class' `GenerateClass` method generates a class called `Vsix`, and marks it `internal`.  The current naming and access modification do not hold up under best practices coding conventions. `{FileNamespace}Manifest` is much more descriptive.

SUGGESTION: Make this user-configurable on a per-project basis.  I am not sure how to do that, only to update the naming and access modifier.

### PR DESCRIPTION
**Pull request by astrohart**
Made the generated `Manifest` class `public`.
Made the generated `Manifest` class not be named `Vsix` but instead, `{FileNamespace}Manifest`.

This is to avoid name collisions for projects such as `ExtensionPackTools` where we have actually three different VSIXs in the same solution and then there are name collisions in shared code.

### CODE REVIEW

File: src/VsixSynchronizer.csproj [hunk 1]
```diff
@@ -84,6 +84,12 @@
     </VSCTCompile>
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Microsoft.VisualStudio.AppDesigner">
```
Comment (madskristensen) [inline src/VsixSynchronizer.csproj]: Where are these consumed?

File: src/Generators/VsixManifestGenerator.cs [hunk 2]
```diff
@@ -36,13 +36,15 @@ private string GenerateClass(VsixManifest manifest)
...
-            sb.AppendLine($"        public const string Name = \"{EscapeString(manifest.Name)}\";");
-            sb.AppendLine($"        public const string Description = @\"{EscapeVerbatimString(manifest.Description)}\";");
-            sb.AppendLine($"        public const string Language = \"{EscapeString(manifest.Language)}\";");
-            sb.AppendLine($"        public const string Version = \"{EscapeString(manifest.Version)}\";");
-            sb.AppendLine($"        public const string Author = \"{EscapeString(manifest.Author)}\";");
-            sb.AppendLine($"        public const string Tags = \"{EscapeString(manifest.Tags)}\";");
+            sb.AppendLine($"        public static {FileNamespace}Manifest Instance {{ get; }} = new {FileNamespace}Manifest();");
+            sb.AppendLine();
+            sb.AppendLine($"        public string Id {{ get; }} = \"{EscapeString(manifest.ID)}\";");
+            sb.AppendLine($"        public string Name {{ get; }} = \"{EscapeString(manifest.Name)}\";");
```
Comment (madskristensen) [inline src/Generators/VsixManifestGenerator.cs]: The reason these are constants is such that they can be used in attributes. Why change it?

Comment (astrohart) [reply src/Generators/VsixManifestGenerator.cs]: They can be left as constants after all, my bad. I was trying to make the generator produce an interface, but it's not necessary to.  I'll undo the changes here.

Comment (astrohart) [reply src/Generators/VsixManifestGenerator.cs]: I've made two new commits.

**Comment by astrohart** [comment]
I cloned this repo before you merged PR #5, is that why appveyor is failing?  It builds on my machine.

**Comment by astrohart** [comment]
I am going to delete my forked repo, re-fork, re-branch, re-clone, and then re-do this PR.

**Pull request closed without merging**